### PR TITLE
fix: Update user guide based on changes to UI

### LINKED ISSUE #2519: Update user guide based on changes to UI
Mainly due to migration from Angular to React

### PR DESCRIPTION
**Pull request by jpav**
Fixes #2519

### CODE REVIEW

**Comment by TovaCohen** [comment]
Hi @jpav - I haven't worked on AtlasMap and Syndesis doc for a while. Melissa Flinn is the writer for Syndesis/Fuse Online now. However, are these updates just for the AtlasMap user guide - or will they need to be carried over to the Syndesis content? If they are just for AtlasMap, I would happy to review this. But if they will need to go to Syndesis doc, then Melissa is the doc person you want to review this.  Also, the diffs show changes to every file because of spaces at the end of lines, so it is hard to find what the actual changes are. Any chance that you can do something so we can see the actual changes easily?

**Comment by jpav** [comment]
Hi, @TovaCohen, my understanding is this is specifically for AtlasMap standalone, so I wouldn't think it was referenced in any way by Syndesis. Regarding Melissa, I don't see an ID for her to use to mark her as a reviewer. And finally, I failed to notice the ending spaces; I will definitely fix that.

**Comment by gashcrumb** [comment]
I've gone ahead and added @MelissaFlynn to the atlasmap org, it'd be good to keep her in the loop on doc changes for sure.

**Comment by jpav** [comment]
I'd actually like to remove the trailing whitespace going forward, but I did find in the 'gear' settings icon under the 'File Changed' tab, you can select to ignore all whitespace in the file comparisons. Let me know if that doesn't work for you.

File: docs/src/main/asciidoc/user-guide/topics/applying-conditions-to-mappings.adoc
Comment (pleacu) [inline docs/src/main/asciidoc/user-guide/topics/applying-conditions-to-mappings.adoc:66]: typo: you cane nter

File: docs/src/main/asciidoc/user-guide/topics/applying-conditions-to-mappings.adoc
Comment (pleacu) [inline docs/src/main/asciidoc/user-guide/topics/applying-conditions-to-mappings.adoc:120]: In the Mapping Details panel, find the field that you want to apply
- Select the bolt icon and select the transformations from the pull-down menu.
- ...

File: docs/src/main/asciidoc/user-guide/topics/exporting-mappings-to-an-atlasmap-catalog-file.adoc
Comment (pleacu) [inline docs/src/main/asciidoc/user-guide/topics/exporting-mappings-to-an-atlasmap-catalog-file.adoc:10]: You can't select a folder to put the .adm file into.  Web-based apps shouldn't wrote into user-defined folders.  It always writes the adm file into the Downloads folder.

File: docs/src/main/asciidoc/user-guide/topics/quickstart.adoc
Comment (pleacu) [inline docs/src/main/asciidoc/user-guide/topics/quickstart.adoc:64]: Or simply drag the selected target field back to the originating source field for the mapping.

**Review by pleacu — CHANGES_REQUESTED** [review]
A couple of changes...

File: docs/src/main/asciidoc/user-guide/topics/exporting-mappings-to-an-atlasmap-catalog-file.adoc
Comment (jpav) [reply docs/src/main/asciidoc/user-guide/topics/exporting-mappings-to-an-atlasmap-catalog-file.adoc:10]: I see now that is correct, but I'm confused. Why do we allow the user to select a folder in VSC and not in standalone?

File: docs/src/main/asciidoc/user-guide/topics/quickstart.adoc
Comment (jpav) [reply docs/src/main/asciidoc/user-guide/topics/quickstart.adoc:64]: Ah, didn't know about this ability. I'll have to update this in a few places.

**Comment by pleacu** [comment]
VSCode has the notion of a workspace built into it.  The stand-alone
AtlasMap is a React web app where the idea is not typically not let people
pick where they want to write data.

**Comment by jpav** [comment]
I understand the reasoning, but I would disagree about the 'typically' portion. There are a myriad websites that allow the user to select where to download/export data. I'll submit a separate issue for this and correct the documentation for how things work now.

File: docs/src/main/asciidoc/user-guide/topics/combine-multiple-source-fields-into-one-target-field.adoc
Comment (TovaCohen) [inline docs/src/main/asciidoc/user-guide/topics/combine-multiple-source-fields-into-one-target-field.adoc:34]:
```suggestion
fields are in the order in which they are expected to appear in the target field.
```

File: docs/src/main/asciidoc/user-guide/topics/combine-multiple-source-fields-into-one-target-field.adoc
Comment (TovaCohen) [inline docs/src/main/asciidoc/user-guide/topics/combine-multiple-source-fields-into-one-target-field.adoc:36]:
```suggestion
If necessary, increase or decrease the index number of a source field to achieve the desired order.
```

File: docs/src/main/asciidoc/user-guide/topics/descriptions-of-atlasmap-icons.adoc
Comment (TovaCohen) [inline docs/src/main/asciidoc/user-guide/topics/descriptions-of-atlasmap-icons.adoc:25]:
```suggestion
There are also several toolbar buttons that allow you to choose what should be displayed, such as data types, sample data previews,
```

File: docs/src/main/asciidoc/user-guide/topics/exporting-mappings-to-an-atlasmap-catalog-file.adoc
Comment (TovaCohen) [inline docs/src/main/asciidoc/user-guide/topics/exporting-mappings-to-an-atlasmap-catalog-file.adoc:7]:
```suggestion
. Click the *AtlasMap* menu and select *Export the current mappings and support files into a catalog (.adm)*
```

File: docs/src/main/asciidoc/user-guide/topics/importing-java-archive-files-into-atlasmap.adoc
Comment (TovaCohen) [inline docs/src/main/asciidoc/user-guide/topics/importing-java-archive-files-into-atlasmap.adoc:51]:
```suggestion
. In the main toolbar, click the *AtlasMap* menu and select *Import a catalog (.adm)*.
```

File: docs/src/main/asciidoc/user-guide/topics/quickstart.adoc
Comment (TovaCohen) [inline docs/src/main/asciidoc/user-guide/topics/quickstart.adoc:71]:
```suggestion
.. To save your work, click the *AtlasMap* menu and select *Export the current mappings and support files into a catalog (.adm)*
```

File: docs/src/main/asciidoc/user-guide/topics/resetting-data-files-in-atlasmap.adoc
Comment (TovaCohen) [inline docs/src/main/asciidoc/user-guide/topics/resetting-data-files-in-atlasmap.adoc:16]:
```suggestion
. In the pop-up menu, select *Reset all mappings and clear all imported documents*.
```

File: docs/src/main/asciidoc/user-guide/topics/separate-one-source-field-into-multiple-target-fields.adoc
Comment (TovaCohen) [inline docs/src/main/asciidoc/user-guide/topics/separate-one-source-field-into-multiple-target-fields.adoc:19]:
```suggestion
. In the *Target* panel, for each field into which you want to map part of the source field's content, navigate to the field and
```

File: docs/src/main/asciidoc/user-guide/topics/separate-one-source-field-into-multiple-target-fields.adoc
Comment (TovaCohen) [inline docs/src/main/asciidoc/user-guide/topics/separate-one-source-field-into-multiple-target-fields.adoc:35]:
```suggestion
If necessary, increase or decrease the indexes next to a target field to achieve the desired order.
```

File: docs/src/main/asciidoc/user-guide/topics/transform-target-data.adoc
Comment (TovaCohen) [inline docs/src/main/asciidoc/user-guide/topics/transform-target-data.adoc:15]: I'm wondering if the Mapping Details panel still has "Sources" as a heading or whether it changed to "Source" like the main panel.

**Comment by TovaCohen** [comment]
Hi @jpav - I did a copy edit and submitted a few comments. Overall, the doc looks great! I did not read the unchanged content - but I did have one comment on something that just caught my eye.

File: docs/src/main/asciidoc/user-guide/topics/transform-target-data.adoc
Comment (jpav) [reply docs/src/main/asciidoc/user-guide/topics/transform-target-data.adoc:15]: It did not. It is plural in Mapping Details.

File: docs/src/main/asciidoc/user-guide/topics/applying-conditions-to-mappings.adoc
Comment (TovaCohen) [inline docs/src/main/asciidoc/user-guide/topics/applying-conditions-to-mappings.adoc:66]: Typo: replace "cane" with "can"

File: docs/src/main/asciidoc/user-guide/topics/separate-one-source-field-into-multiple-target-fields.adoc
Comment (TovaCohen) [inline docs/src/main/asciidoc/user-guide/topics/separate-one-source-field-into-multiple-target-fields.adoc:35]: Typo:  replace "of" with "or"

**Comment by TovaCohen** [comment]
Hi @jpav, I found two typos. Otherwise, this LGTM!

**Comment by jpav** [comment]
Thanks, @TovaCohen. I've corrected those errors. If you could approve the PR, I'll get it checked in.

**Review by TovaCohen — APPROVED** [review]
LGTM!

File: docs/src/main/asciidoc/user-guide/topics/exporting-mappings-to-an-atlasmap-catalog-file.adoc
Comment (MelissaFlinn) [inline docs/src/main/asciidoc/user-guide/topics/exporting-mappings-to-an-atlasmap-catalog-file.adoc:16]: change "the user's" to "your"

File: docs/src/main/asciidoc/user-guide/topics/exporting-mappings-to-an-atlasmap-catalog-file.adoc
Comment (MelissaFlinn) [inline docs/src/main/asciidoc/user-guide/topics/exporting-mappings-to-an-atlasmap-catalog-file.adoc:18]: change "via" to "by using"

File: docs/src/main/asciidoc/user-guide/topics/quickstart.adoc
Comment (MelissaFlinn) [inline docs/src/main/asciidoc/user-guide/topics/quickstart.adoc:59]: do you need to delete the ",or" at the end of this line?

**Review by MelissaFlinn — COMMENTED** [review]
Sorry it has taken me so long to reply. 
Like Tova, I scanned through the changed text only. 
I added a few comments about minor things that could be changed. 
I agree - your tip about hiding the white spaces was very helpful. :)

**Comment by jpav** [comment]
Thanks, @MelissaFlinn, those were good catches. Your turn to approve changes :-).

**Review by TovaCohen — APPROVED** [review]
LGTM again :-)

**Comment by jpav** [comment]
@pleacu, Looks like this is waiting for your approval now.

**Review by MelissaFlinn — APPROVED** [review]
Sorry it has taken me so long to reply. 
Like Tova, I scanned through the changed text only. 
I added a few comments about minor things that could be changed. 
I agree - your tip about hiding the white spaces was very helpful. :)

**Review by pleacu — APPROVED** [review]
+1 - tech doc approves.

**Pull request merged**
